thrust/thrust/system/cuda/detail/core/util.h

```diff
@@ -428,7 +365,7 @@ THRUST_RUNTIME_FUNCTION typename get_plan<Agent>::type get_agent_plan(int ptx_ve
 }
 
 // XXX keep this dead-code for now as a gentle reminder
-//     that kernel luunch which reats plan values is the most robust
+//     that kernel launch which reats plan values is the most robust
 //     mechanism to extract sm-specific tuning parameters
```

Comment:
I actually wonder whether there are more typos in this comment or my English is just not good enough. Could be `treats plain values`.